runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -2497,6 +2497,15 @@ private static String longBossName(String boss)
 			case "vard awakened":
 				return "Vardorvis (awakened)";
 
+			// lunar chest variants
+			case "lunar chests":
+			case "perilous moons":
+			case "perilous moon":
+			case "moons of peril":
+			case "mop":
+			case "lc":
```

Comment:
I understand these are abbreviations for Moons of Peril and Lunar Chest, but I don't think anyone uses them. I'm inclined to stick to just the lines above and remove these.
```suggestion
```